components/eam/docs/user-guide/index.md

```diff
@@ -1 +1,139 @@
+ linoz_data_type                = 'INTERP_MISSING_MONTHS'
+```
+
+Refer to [this page](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/3764486280/Production+of+the+Linoz+v3+data) for more details on Linoz v3 input files. 
+
+#### Topography
+The global elevation on the atmosphere grid is a key input dataset.  The dataset is grid dependent. It contains the geopotential data (on both the GLL dynamics and PG2 physics grids) and two surface roughness quantities, `SGH` and `SGH30`.
+
+EAMv3 NE30 data:  
+```
+/lcrc/group/e3sm/data/inputdata/atm/cam/topo/USGS-gtopo30_ne30np4pg2_x6t-SGH.c20210614.nc'
+```
+
+This file is computed via a complex procedure that starts with high resolution dataset (for EAMv3, we use the GTOPO30, a 30 arc-second resolution data set on a lat-lon grid) that is then downsampled to a 3km cubed-sphere grid (cube3000) and then downsampled to the atmosphere  grid on the (GLL nodes), and then smoothed with the same viscosity operator used by the dycore. The smoothed GLL topography is then mapped to the PG2 grid. Finally, two different surface roughness fields are computed:
+
+- `SGH30`: the variance between GTOPO30 and GTOPO30-downsampled-to-PG2 (independent of any dycore specific smoothing).     (used by CLUBB, TMS, vertical_diffusion)
+- `SGH`:  the variance between the cube3000 data and the smoothed PG2 data.  (used by GWD parameterizations)
+
+#### Land-use / land cover change
+
+- <span style="color:red">Info needed on land-use land cover change / land surface data</span>
+
+#### Ocean/sea ice
+
+- <span style="color:red">SST file</span>
+- <span style="color:red">Sea ice fraction</span>
+
+#### Solar input
+
+As with greenhouse gas emissions, solar input files are taken from the input4MIPs data collection that were prepared for CMIP6 Forcing Datasets.
+
+```
+inputdata/atm/cam/solar/Solar_1850-2299_input4MIPS_c20181106.nc
+```
+
+### Specifying output (history files)
+
+These are specified in the `user_nl_eam`. 
+
+By default, EAM will output a set of monthly-averaged variables. Additional output files can be specified using the following flags:
+
+`finclX` - List of variables (in single quotes and separated by commas) that are added to tape X.
+
+`fexclX` - List of variables (in single quotes and separated by commas) that will be excluded in tape X.
+
+`nhtfrq` - List of write frequencies for each of the history files. A value of 0 denotes a monthly frequency. Negative values denote hourly frequencies (e.g., `-3` will write an output every 3 hours). Positive values denotes the frequency in model timesteps (e.g., `4` will write an output every 4 timesteps).
+
+`mfilt` - List that sets the number of timesteps to write in a single file before starting a new file.
+
+`avgflag_pertape` - List that sets the type of output to write. Choices are `'A'` for time-averaged output, `'A'` for instantaneous output, `'MIN'` for time-minimum output, and `'MAX'` for time-maximum output. 
+
+
+#### Example output specification:
+
+```
```

Comment:
This code block didn't get rendered correctly at https://e3sm-project.github.io/E3SM/pr-preview/pr-6314/EAM/user-guide/#example-output-specification